records/generator.go

```diff
@@ -436,7 +442,10 @@ func (rg *RecordGenerator) listenerRecord(listener string, ns string) {
 
 func (rg *RecordGenerator) taskRecords(sj state.State, domain string, spec labels.Func, ipSources []string) {
 	for _, f := range sj.Frameworks {
-		enumerableFramework := &EnumerableFramework{Name: f.Name}
+		enumerableFramework := &EnumerableFramework{
+			Name: f.Name,
+			Tasks: []*EnumerableTask{},
```

Comment:
can't you just use a json `omitempty` tag for `Tasks` in the struct where it's defined?